src/axom/quest/MarchingCubes.cpp

```diff
@@ -206,31 +206,43 @@ void MarchingCubesSingleDomain::allocateImpl()
   if(m_runtimePolicy == RuntimePolicy::seq)
   {
     m_impl = m_ndim == 2
-      ? std::unique_ptr<ImplBase>(new MarchingCubesImpl<2, axom::SEQ_EXEC>)
-      : std::unique_ptr<ImplBase>(new MarchingCubesImpl<3, axom::SEQ_EXEC>);
+      ? std::unique_ptr<ImplBase>(
+          new MarchingCubesImpl<2,
+                                axom::SEQ_EXEC,
+                                axom::execution_space<axom::SEQ_EXEC>::loop_policy>)
+      : std::unique_ptr<ImplBase>(
+          new MarchingCubesImpl<3,
+                                axom::SEQ_EXEC,
+                                axom::execution_space<axom::SEQ_EXEC>::loop_policy>);
   }
 #ifdef _AXOM_MC_USE_OPENMP
   else if(m_runtimePolicy == RuntimePolicy::omp)
   {
     m_impl = m_ndim == 2
-      ? std::unique_ptr<ImplBase>(new MarchingCubesImpl<2, axom::OMP_EXEC>)
-      : std::unique_ptr<ImplBase>(new MarchingCubesImpl<3, axom::OMP_EXEC>);
+      ? std::unique_ptr<ImplBase>(
+          new MarchingCubesImpl<2, axom::OMP_EXEC, RAJA::seq_exec>)
+      : std::unique_ptr<ImplBase>(
+          new MarchingCubesImpl<3, axom::OMP_EXEC, RAJA::seq_exec>);
```

Comment:
```suggestion
          new MarchingCubesImpl<3, axom::OMP_EXEC, axom::SEQ_EXEC>);
```

Reply:
Done, but note that this requires a corresponding change in the third template argument of `MarchingCubesImpl`, since it is changed from a loop policy to an execution space.